### PR TITLE
Org vertical

### PR DESCRIPTION
**Pull request by haoyxc**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export SENTRY_URL="https://d24382d743314dadadb80d1eabdb139e@sentry.pennlabs.org/13"
```
Comment (ezwang) [inline .env]: Don't commit this file! Add it to the `.gitignore` and ensure that it is removed before merging.

File: components/Form.js [hunk 2]
```diff
@@ -25,6 +27,19 @@ class Form extends React.Component {
...
+    // console.log('HANDLE UPLOAD', e)
+    if (e.target.files[0]) {
+      this.setState({
+        hasImg: true,
+        imgName: e.target.files[0].name,
+      })
+    }
+
+    // console.log(e.target.files[0].name, 'woo')
+    // console.log(e.currentTarget, 'CT')
```
Comment (ezwang) [inline components/Form.js]: Remove these comments before merging!

File: package.json [hunk 3]
```diff
@@ -21,12 +21,13 @@
...
     "react-ga": "^2.6.0",
     "react-lazy-load": "^3.0.13",
+    "react-router-dom": "^5.1.2",
     "react-select": "^3.0.4",
     "styled-components": "^4.3.2"
   },
   "scripts": {
     "lint": "eslint \"./{pages,components}/**/*.js\" \"*.js\"",
-    "dev": "node server/index.js",
+    "dev": "nodemon server/index.js",
```
Comment (ezwang) [inline package.json]: If you make this change, ensure that `nodemon` is installed in the `package.json` as a developer dependency.

File: pages/club_tree_example.json [hunk 4]
```diff
@@ -0,0 +1,59 @@
+{
```
Comment (ezwang) [inline pages/club_tree_example.json]: Remove this file before merging!

File: static/img/icons/success.svg [hunk 5]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (ezwang) [inline static/img/icons/success.svg]: Just to make sure, is this from [Feather Icons](https://feathericons.com)?

File: .gitignore [hunk 6]
```diff
@@ -1,5 +1,5 @@
 node_modules/
 .next/
-env.sh
```
Comment (ezwang) [inline .gitignore]: You might want to keep both of these entries! Some people use `.env`, and others use a `env.sh` file.

File: frontend/components/Form.js [hunk 7]
```diff
@@ -257,6 +283,14 @@ class Form extends Component {
               <span className="file-label">Choose a file...</span>
             </span>
           </label>
+          {this.state.hasImg ? (
+            <span>
+              {' '}
+              <Icon name="check-circle-green" size="1.8rem" alt="checkbox"></Icon>
+              {'   '}
```
Comment (cacabo) [inline frontend/components/Form.js]: More than one space won't make a difference
```suggestion
              {' '}
```

File: frontend/components/OrgPage/ChildDisplay.js [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+    background: ${LIGHT_GRAY}
+    color: ${BLACK}
+  }
+`
+const Text = s.p`
+  color: ${BLACK};
+  font-size: 16px;
+
+  &:hover{
+  }
```
Comment (cacabo) [inline frontend/components/OrgPage/ChildDisplay.js]: should be deleted

File: frontend/components/OrgPage/ChildDisplay.js [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+`
+const TextLink = s.a`
+  color: ${BLACK};
+  font-size: 16px;
+
+  &:hover{
+    font-weight: bold;
+    
+  }
```
Comment (cacabo) [inline frontend/components/OrgPage/ChildDisplay.js]: Think this is kinda an odd UX. Would prefer a change in color. Also if the default link is black I'm not sure it's intuitive to the user that this is actually a link

File: frontend/components/OrgPage/ChildDisplay.js [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+`
+
+export default class ChildDisplay extends Component {
```
Comment (cacabo) [inline frontend/components/OrgPage/ChildDisplay.js]: Can this be made a functional component?

File: frontend/components/OrgPage/EditBox.js [hunk 11]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export default function EditBox() {
```
Comment (cacabo) [inline frontend/components/OrgPage/EditBox.js]:
```suggestion
export default = () => {
```

File: frontend/components/OrgPage/EditBox.js [hunk 12]
```diff
@@ -0,0 +1,9 @@
...
+  return (
+    <div>
+      <p>Edit</p>
+    </div>
+  );
```
Comment (cacabo) [inline frontend/components/OrgPage/EditBox.js]: Make sure you have prettier set up
```suggestion
  )
```

File: frontend/components/OrgPage/OrgChildren.js [hunk 13]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import ChildDisplay from './ChildDisplay'
+import '../../constants/colors'
+import s from 'styled-components'
```
Comment (cacabo) [inline frontend/components/OrgPage/OrgChildren.js]: package imports should come before relative ones

File: frontend/components/OrgPage/OrgChildren.js [hunk 14]
```diff
@@ -0,0 +1,26 @@
...
+  return (
+    <div
+      style={{
+        whiteSpace: 'pre-wrap',
+        margin: '0px',
+      }}
+    >
+      {children.map(c => {
+        return <ChildDisplay child={c}></ChildDisplay>
+      })}
```
Comment (cacabo) [inline frontend/components/OrgPage/OrgChildren.js]: Needs a `key` prop passed to `ChildDisplay`

File: frontend/components/OrgPage/OrgLinks.js [hunk 15]
```diff
@@ -0,0 +1,7 @@
+import React, { Component } from 'react'
+
+export default class OrgLinks extends Component {
```
Comment (cacabo) [inline frontend/components/OrgPage/OrgLinks.js]: Do we need this file? If so, can it be refactored into a functional component?

File: frontend/pages/org.js [hunk 16]
```diff
@@ -0,0 +1,93 @@
+import renderPage from '../renderPage.js'
+import { getDefaultClubImageURL, doApiRequest } from '../utils'
+import React, { Component } from 'react'
```
Comment (cacabo) [inline frontend/pages/org.js]: Package import should come before static ones

File: frontend/pages/org.js [hunk 17]
```diff
@@ -0,0 +1,93 @@
...
+              }}
+            ></div>
+            {/* {console.log(club)} */}
+            {/* <OrgTabs club={club} /> */}
+          </div>
+          <div className="column is-8">
+            <OrgTabs club={club} children={children} />
+            {console.log(children)}
+            {/* <OrgChildren children={children}></OrgChildren> */}
+            {/* <Image src={club.image_url || getDefaultClubImageURL()} /> */}
```
Comment (cacabo) [inline frontend/pages/org.js]: Remove commented out code

File: frontend/pages/org.js [hunk 18]
```diff
@@ -0,0 +1,93 @@
...
+          <div className="column is-4">
+            <InfoBox club={club} />
+            {/* <SocialIcons club={club} /> */}
+            <div
+              dangerouslySetInnerHTML={{
+                __html: club.description || EMPTY_DESCRIPTION,
+              }}
+            ></div>
+            {/* {console.log(club)} */}
+            {/* <OrgTabs club={club} /> */}
```
Comment (cacabo) [inline frontend/pages/org.js]: Remove commented out code

File: frontend/pages/org.js [hunk 19]
```diff
@@ -0,0 +1,93 @@
...
+          club={club}
+          userInfo={this.props.userInfo}
+          favorites={this.props.favorites}
+          updateFavorites={this.props.updateFavorites}
+        />
+        <div></div>
+        <div className="columns">
+          <div className="column is-4">
+            <InfoBox club={club} />
+            {/* <SocialIcons club={club} /> */}
```
Comment (cacabo) [inline frontend/pages/org.js]:
```suggestion
```

File: frontend/pages/org.js [hunk 20]
```diff
@@ -0,0 +1,93 @@
...
+            {/* <OrgChildren children={children}></OrgChildren> */}
+            {/* <Image src={club.image_url || getDefaultClubImageURL()} /> */}
+          </div>
+        </div>
+      </div>
+    )
+  }
+}
+
+Org.getInitialProps = async props => {
```
Comment (cacabo) [inline frontend/pages/org.js]:
```suggestion
Org.getInitialProps = async ({ query }) => {
```

File: frontend/pages/org.js [hunk 21]
```diff
@@ -0,0 +1,93 @@
...
+  const { query } = props
+  return { query: query }
```
Comment (cacabo) [inline frontend/pages/org.js]:
```suggestion
  return { query }
```

File: frontend/pages/org.js [hunk 22]
```diff
@@ -0,0 +1,93 @@
...
+      <div style={{ padding: '30px 50px' }}>
+        <Header
+          club={club}
+          userInfo={this.props.userInfo}
+          favorites={this.props.favorites}
+          updateFavorites={this.props.updateFavorites}
+        />
+        <div></div>
+        <div className="columns">
+          <div className="column is-4">
```
Comment (cacabo) [inline frontend/pages/org.js]: Seems like there might be some duplicate code here. @ezwang do you think it's worth refactoring this type of stuff into a component used across several pages or no?

File: frontend/pages/settings.js [hunk 23]
```diff
@@ -3,14 +3,21 @@ import { BORDER_RADIUS } from '../constants/measurements'
...
 
 import renderPage from '../renderPage'
-import { doApiRequest, formatResponse } from '../utils'
+import {
+  doApiRequest,
+  formatResponse,
+  API_BASE_URL,
+  ROLE_OFFICER,
+} from '../utils'
 import s from 'styled-components'
```
Comment (cacabo) [inline frontend/pages/settings.js]: Package imports should come before relative ones

File: frontend/public/static/img/icons/success.svg [hunk 24]
```diff
@@ -0,0 +1,38 @@
...
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
```
Comment (cacabo) [inline frontend/public/static/img/icons/success.svg]: Does the SVG need these empty tags to render correctly? Can you try deleting them and seeing if it still looks ok?

**Comment by cacabo** [comment]
Hey @haoyxc let us know if you need any help with anything or when you want another review of your PR!

**Comment by ezwang** [comment]
Replaced by https://github.com/pennlabs/penn-clubs/pull/114.

**Pull request closed without merging**